src/bittrade/data/google_client.py

```diff
@@ -0,0 +1,101 @@
+"""Module to retrieves Google Finance data.
+
+This module contains various price for stocks and crypto.
+
+Todo:
+    * EMPTY
+    * You have to also use ``sphinx.ext.todo`` extension
+"""
+from datetime import datetime
+
+import pandas as pd
+import requests
+
+
+def get_price_data(query):
+    """
+    Use Google Finance to get data.
```

Comment:
see[ here](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for style of doc string